chat-plugins/scavengers.js

```diff
@@ -928,7 +928,7 @@ let commands = {
 		points = parseInt(points);
 
 		if (!targetId || targetId === 'constructor' || targetId.length > 18) return this.errorReply("Invalid username.");
-		if (!points || points < 0) return this.errorReply("Points must be an integer greater than 0.");
+		if (!points || points < 0 || points === Infinity) return this.errorReply("Points must be a non-Infinity integer greater than 0.");
```

Comment:
Also "non-Infinity" is a silly way to say "finite".

Even so, "finite integer" is a silly way to say "integer".